python/mlc_chat/compiler_pass/pipeline.py

```diff
@@ -34,13 +35,34 @@ def transform_module(self, mod: IRModule, _ctx: tvm.transform.PassContext) -> IR
         return mod
 
 
+@tvm.transform.module_pass(opt_level=0, name="DebugDump")
+class _DebugDump:  # pylint: disable=too-few-public-methods
+    """A dummy compiler pass that does nothing but logging.
+    Only enabled when debug_dump is not None"""
+
+    def __init__(self, file_name: str, file_path: Optional[str], show_meta: bool = False):
```

Comment:
Also its type is `Optional[Path]`

```suggestion
    def __init__(self, file_name: str, file_path: Optional[Path], show_meta: bool = False):
```